tests/test_utils.py

```diff
@@ -366,6 +367,15 @@ def test_is_probability(self):
         not_probabilities = np.array([-1.1, 0.3, 0.7])
         self.assertFalse(is_probability(not_probabilities))
 
+    def test_check_pytorch_loss_param(self):
+        import torch.nn as nn
+
+        loss = nn.CrossEntropyLoss()
+        self.assertFalse(check_pytorch_loss_param(loss))
+
+        loss = nn.PoissonNLLLoss()
+        self.assertTrue(check_pytorch_loss_param(loss))
+
```

Comment:
Are these tests still required with the currently implemented solution of listing all loss functions?

Reply:
Removed.